newwebsite/generate.jl

```diff
@@ -0,0 +1,605 @@
+### A Pluto.jl notebook ###
+# v0.17.2
+
+using Markdown
+using InteractiveUtils
+
+# This Pluto notebook uses @bind for interactivity. When running this notebook outside of Pluto, the following 'mock version' of @bind gives bound variables a default value (instead of an error).
+macro bind(def, element)
+    quote
+        local iv = try Base.loaded_modules[Base.PkgId(Base.UUID("6e696c72-6542-2067-7265-42206c756150"), "AbstractPlutoDingetjes")].Bonds.initial_value catch; b -> missing; end
+        local el = $(esc(element))
+        global $(esc(def)) = Core.applicable(Base.get, el) ? Base.get(el) : iv(el)
+        el
+    end
+end
+
+# ╔═╡ 3f2428bc-4ba4-11ec-3f1b-1326777bfdd2
+import Pkg
+
+# ╔═╡ 257b4034-3ef7-4a7b-8384-c8d25e618ae2
+Pkg.activate(".")
```

Comment:
```suggestion
Pkg.activate("."); Pkg.instantiate()
```

Reply:
I haven't fully understood when instantiating is necessary - but this gave me an error because I didn't have all packages installed

Reply:
Thanks! It never hurst to do it